pages/api/models/selfCheckQuestion.ts

```diff
@@ -0,0 +1,43 @@
+import { model, Schema, Types } from "mongoose";
+
+enum QuestionType {
+    MULTIPLE_CHOICE = "multiple_choice",
+    MULTI_SELECT = "multi_select",
+}
+
+interface SelfCheckQuestionInterface {
+    _id: Types.ObjectId;
+    type: QuestionType;
+    question: string;
+    options: [Schema.Types.Mixed];
+    questionNumber: number;
+}
+
+const SelfCheckQuestionSchema = new Schema<SelfCheckQuestionInterface>(
+    {
+        type: {
+            type: QuestionType,
```

Comment:
For enum types, you want String as the type with the enum field.
```suggestion
            type: String,
            enum: Object.values(QuestionType)
```
See: https://mongoosejs.com/docs/schematypes.html#schematype-options  
https://mongoosejs.com/docs/validation.html#custom-validators